go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/husniadil/uuid
 go 1.15
```

Comment:
Ensure that you run your test suite after this dependency update to verify that no breaking changes have affected your code. It is crucial to validate that the new version of the library behaves as expected in your application context.